node1_processtime: 0.00 # millisecond

point_by_point_z: -1.6 # meter
point_by_point_r: 5.0 # meter
point_by_point_tail_theta: 45.0 # degree
point_by_point_tail_length: 20.0 # meter

clustering_tolerance: 1.0 # meter
clustering_size_min: 30 # points number
clustering_size_max: 1000000 # points number